pkg/probe/probes.go

```diff
@@ -27,13 +28,34 @@ var (
 	log = logf.Log.WithName("probe")
 )
 
+type Probe struct {
+	name    string
+	timeout time.Duration
+	run     func(client.Client, time.Duration) error
+}
+
 const (
 	defaultGwRetrieveTimeout  = 120 * time.Second
 	defaultGwProbeTimeout     = 120 * time.Second
+	defaultDnsProbeTimeout    = 120 * time.Second
 	apiServerProbeTimeout     = 120 * time.Second
 	nodeReadinessProbeTimeout = 120 * time.Second
 )
 
+func currentStateAsGJson() (gjson.Result, error) {
+	observedStateRaw, err := nmstatectl.Show()
+	if err != nil {
+		return gjson.Result{}, errors.Wrap(err, "failed retrieving current state")
```

Comment:
i think you can switch  "github.com/pkg/errors" with "errors" ?

Reply:
You mean using go 1.13 "%w" to wrap errors ?

Reply:
this is an option as well, if it suits,
but i meant to replace the import line

Reply:
`errors` and `github.com/pkg/errors` are not the same package they have different interfaces.